nestkernel/spike_data.h

```diff
@@ -33,6 +33,14 @@
 namespace nest
 {
 
+enum enum_status_spike_data_id
```

Comment:
Is this used anywhere?

Reply:
yes, e.g., https://github.com/nest/nest-simulator/blob/a406662974b459d875aedf8fdea749b367ce09ae/nestkernel/spike_data.h#L145